resources/benchmark-runner.mjs

```diff
@@ -352,7 +352,9 @@ export class BenchmarkRunner {
         const runSync = () => {
             if (params.warmupBeforeSync) {
                 const startTime = performance.now();
-                while (performance.now() - startTime < params.warmupBeforeSync); // Infinite loop for the specified ms.
+                // Infinite loop for the specified ms.
+                while (performance.now() - startTime < params.warmupBeforeSync)
+                    continue;
```

Comment:
Why are we making this change?

Reply:
Auto-formatting changed it to:

```
while (performance.now() - startTime < params.warmupBeforeSync) // Infinite loop for the specified ms.
    ;
```

So I preferred moving the comment + adding a `continnue` instead of having this weird dangling semicolon.